requirements.txt

```diff
@@ -1,3 +1,2 @@
-beautifulsoup4
-lxml
-requests
+beautifulsoup4==4.14.2
```

Comment:
Why would we want to pin the dependencies down to the micro version? If we let them flap completely, or pinned them down to a major or minor version, we wouldn't need to apply changes to this file when a security update was shipped which incremented the micro version.